src/core/textures/Texture.js

```diff
@@ -141,6 +141,12 @@ export default class Texture extends EventEmitter
          */
 
         this._updateID = 0;
+
+        /**
+         * Extra field for extra plugins. May contain clamp settings and some matrices
+         * @type {Object}
+         */
+        this.transform = null;
```

Comment:
Its there in case if user wants to specify clamping constants and stuff before he creates TilingSprite